archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -217,6 +217,9 @@ package_clang-git() {
 
     # Move analyzer scripts out of /usr/libexec
     mv "$pkgdir"/usr/libexec/{ccc,c++}-analyzer "$pkgdir/usr/lib/clang/"
+    mv "$pkgdir"/usr/libexec/analyze-{c++,cc} "$pkgdir/usr/lib/clang/"
+    mv "$pkgdir"/usr/libexec/intercept-{c++,cc} "$pkgdir/usr/lib/clang/"
+
     rmdir "$pkgdir/usr/libexec"
     sed -i 's|libexec|lib/clang|' "$pkgdir/usr/bin/scan-build"
```

Comment:
And if you have a locally compiled version, you might want to check if there's any new references to libexec that needs to be patched up now.

Reply:
nothing new

Reply:
ok